docs/src/pages/components/Components.jsx

```diff
@@ -156,7 +156,7 @@ function Components() {
             { text: "Facebook", href: "http://www.google.com" },
             { text: "Instagram", href: "http://www.google.com" },
           ]}
-          copyright="© DXC Technology 2020. All rights reserved."
+          copyright="© DXC Technology 2022. All rights reserved."
```

Comment:
Remove this line. This Footer (and every other one in our CDK must use the preset value).

Reply:
Done